docs/execution_nodes.md

```diff
@@ -0,0 +1,83 @@
+
+Below is an example of a single AWX pod connecting to two different execution nodes. For each execution node, the awx-ee container makes an outbound TCP connection to the machine via Receptor.
+
+```
+                                 AWX POD
+                             ┌──────────────┐
+                             │              │
+                             │ ┌──────────┐ │
+┌─────────────────┐          │ │ awx-task │ │
+│ execution node 1│◄────┐    │ ├──────────┤ │
+├─────────────────┤     ├────┼─┤ awx-ee   │ │
+│ execution node 2│◄────┘    │ ├──────────┤ │
+└─────────────────┘ Receptor │ │ awx-web  │ │
+                      TCP    │ └──────────┘ │
+                     Peers   │              │
+                             └──────────────┘
+```
+
+Note, if the AWX deployment is scaled up, the new AWX pod will also make TCP connections to each execution node.
+
+
+## Overview
+Adding an execution instance involves a handful of steps:
+
+1. Start a machine that is accessible from the k8s cluster (Red Hat family of operating systems are supported)
+2. Create a new AWX Instance with `hostname` being the IP or DNS name of your remote machine.
+3. Download the install bundle for this newly created instance.
+4. Enable upstream Receptor repository on the machine.
+5. Run the install bundle playbook against your remote machine.
+6. Wait for the instance to report a Ready state. Now jobs can run on that instance.
+
+
+### 1. Start machine
+
+Bring a machine online with a compatible Red Hat family OS (e.g. RHEL 8 and 9). This machines needs a static IP, or a resolvable DNS hostname that the AWX cluster can access. The machine will also need an available open port to establish inbound TCP connections on (default is 27199).
+
+In general the more CPU cores and memory the machine has, the more jobs that can be scheduled to run on that machine at once. See https://docs.ansible.com/automation-controller/4.2.1/html/userguide/jobs.html#at-capacity-determination-and-job-impact for more information on capacity.
+
+
+### 2. Create Instance in AWX
+
+Use the UI Instance page or `api/v2/instances` endpoint to add a new instance.
+- `hostname` ("Name" in UI) is the IP address or DNS name of your machine.
+- `node_type` is "execution"
+- `node_state` is "installed"
+- `listener_port` is an open port on the remote machine used to establish inbound TCP connections.
+
+
+### 3. Download the install bundle
+
+On the Instance Details page, click Install Bundle and save the *tar.gz file to your local computer and extract contents.
+
+
+### 4. Enable repository for Receptor
```

Comment:
i think it flows better if we fold this into 1? since they are both taking about preparing machine for remote execution node